src/test/fixtures/profiles/processed-profile.js

```diff
@@ -76,76 +76,23 @@ function _refineMockPayload(
 
 export function addMarkersToThreadWithCorrespondingSamples(
   thread: Thread,
-  markers: TestDefinedMarkers,
-  interval: Milliseconds
+  markers: TestDefinedMarkers
 ) {
   const stringTable = thread.stringTable;
   const markersTable = thread.markers;
 
-  const allTimes = new Set();
-
   markers.forEach(([name, time, data]) => {
     markersTable.name.push(stringTable.indexForString(name));
     markersTable.time.push(time);
     markersTable.data.push(_refineMockPayload(data));
     markersTable.category.push(0);
     markersTable.length++;
-
-    // Try to get a consistent profile containing all markers
```

Comment:
The code here was adding samples before and after all markers added in this function, just so that the range included them. This is now unneeded with the new range computation.